src/login/logind.c

```diff
@@ -1171,9 +1171,23 @@ int main(int argc, char *argv[]) {
          * existence of /run/systemd/seats/ to determine whether
          * logind is available, so please always make sure this check
          * stays in. */
-        mkdir_label("/run/systemd/seats", 0755);
-        mkdir_label("/run/systemd/users", 0755);
-        mkdir_label("/run/systemd/sessions", 0755);
+        r = mkdir_label("/run/systemd/seats", 0755);
```

Comment:
> systemd-tmpfiles creates /run/systemd/{seats,users,sessions} on startup
systemd-logind fails:....

Ah, yes, didn't know. `mkdir_` prefix looks confusing in this case.